tensorflow/python/ops/custom_gradient.py

```diff
@@ -201,6 +201,10 @@ def second_order_and_transpose(ddy):
   Returns:
     A function `h(x)` which returns the same value as `f(x)[0]` and whose
     gradient (as calculated by `tf.gradients`) is determined by `f(x)[1]`.
+  
+  Raises:
+    RuntimeError: tf.gradients is not supported when eager execution is
+      enabled. Use tf.GradientTape instead.
```

Comment:
Suggestion (nit): maybe we could use backticks here, so that the API references (`tf.gradients`) can be auto-linked during build for tensorflow . org.

e.g.

```suggestion
    RuntimeError: `tf.gradients` is not supported when eager execution is
      enabled. Use `tf.GradientTape` instead.
```

(https://www.tensorflow.org/community/contribute/docs_style#write_about_code)

cc @lamberta @MarkDaoust

Reply:
@8bitmp3 Thanks. Updated the PR with your suggestion.